src/FFmpegReader.cpp

```diff
@@ -1593,11 +1603,19 @@ void FFmpegReader::ProcessAudioPacket(int64_t requested_frame) {
 
 		// determine how many samples were decoded
 		int plane_size = -1;
+#if HAVE_CH_LAYOUT
+		data_size = av_samples_get_buffer_size(&plane_size, AV_GET_CODEC_ATTRIBUTES(aStream, aCodecCtx)->ch_layout.nb_channels,
```

Comment:
Since we do this calculation on both code paths, it might be nice to set the `data_size` and `nb_samples` in the conditional #if, and then do the `packet_samples` calculation 1 time, below the conditional ifs.